### PR TITLE
#162 Magazine-Issue: Fixed scroll action on explore-this-issue button click

### LINKED ISSUE #162: Magazine: "Explore this issue" button does not work
image

e.g. 
https://main--24life--hlxsites.hlx.live/magazine/volume-6-issue-2

### PR DESCRIPTION
**Pull request by iamitshri**
Fix #162

scroll to article: 

https://162-magazine-explore-button--24life--hlxsites.hlx.page/magazine/volume-3-issue-3

video as section's next sibling: https://162-magazine-explore-button--24life--hlxsites.hlx.page/magazine/volume-5-issue-2

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -2,3 +2,4 @@
 <script src="/scripts/lib-franklin.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
 <link rel="stylesheet" href="/styles/styles.css"/>
+<link rel="alternate" type="application/rss+xml" title="24Life &raquo; Feed" href="https://www.24life.com/feed/" />
```
Comment (iamitshri) [inline head.html:5]: This change was required for adding rss feed to all pages

File: blocks/magazine-hero/magazine-hero.js [hunk 2]
```diff
@@ -56,13 +56,9 @@ function createLeftOverlay(overlay, config) {
...
 
-function scrollToFirstParagraph() {
-  for (const p of document.querySelectorAll('.default-content-wrapper p')) {
-    if (!p.querySelector('iframe')) {
-      p.scrollIntoView({ behavior: 'smooth' });
-      break;
-    }
-  }
+function scrollToNext(event) {
+  const heroContainer = event.target.closest('.magazine-hero-container');
```
Comment (synox) [inline blocks/magazine-hero/magazine-hero.js:60]: This skips the quote on https://162-magazine-explore-button--24life--hlxsites.hlx.page/magazine/volume-3-issue-3

how about 
```js
block.closest('.magazine-hero-wrapper').nextElementSibling || block.closest('. section).nextElementSibling

File: tools/importer/import_article.js [hunk 3]
```diff
@@ -69,18 +69,9 @@ export default {
       // magazine article e.g. https://www.24life.com/make-2019-the-year-you-dont-get-hurt/
       await detectMagazineHero(params, magazineSection, main, document);
     } else {
-      // currently we only import magazine articles. If something else is detected, that is an error.
+      // currently we only import magazine articles.
+      // If something else is detected, that is an error.
       throw new Error('not a magazine article');
-
-      // start with h1, then image
```
Comment (synox) [inline tools/importer/import_article.js:75]: please keep the code, just comment it out.

File: blocks/magazine-hero/magazine-hero.js
Comment (synox) [inline blocks/magazine-hero/magazine-hero.js:1]: Please try adding `div` to https://github.com/hlxsites/24life/blob/ba5dc963be9880f88bd417017dc8b4dd7b585017/styles/styles.css#L154-L158 and if this improves the scroll position

**Review by synox — APPROVED** [review]

**Pull request merged**